src/mito2/src/manifest/action.rs

```diff
@@ -62,13 +62,57 @@ pub struct RegionRemove {
     pub region_id: RegionId,
 }
 
-/// Last data truncated in the region.
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
+#[derive(Serialize, Clone, Debug, PartialEq, Eq)]
 pub struct RegionTruncate {
     pub region_id: RegionId,
     pub kind: TruncateKind,
 }
```

Comment:
Can internal tagging solve this? We use the old kind as the default type.
https://serde.rs/enum-representations.html#internally-tagged